src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -109,3 +109,19 @@ object OTreeIndex {
   }
 
 }
+
+case class EmptyIndex() extends FileIndex {
```

Comment:
Looks like a singleton, probably it make sense to implement it as object.

Reply:
Done in 0028ee4